src/components/ui/common/Actions.js

```diff
@@ -44,5 +45,22 @@ exports.uncheckAllDropdownOptions = function(opts) {
     openBtn.click();
 
     opts.base.all(by.css('[checked=checked]')).click();
+    openBtn.click();
 
+};
+//This function opens the drop down list But does not closes it!
+//so you can perform actions such as getText while the elements are visible.
+/**
+ * @description gets all dropdown options that are selected
+ *
+ * @param {object} multiSelectElement is the the multi select element
+ *
+ */
+exports.getSelectedOptions = function(multiSelectElement) {
+    multiSelectElement.element(by.css('.msArrow')).click();
+    return multiSelectElement.all(by.css('li')).filter(function(item){
+        return item.all(by.css('input[checked=checked]')).count().then(function(count){
```

Comment:
1) why do we need `all` ? 
2) I think we can simplify to something like `return $('input').isSelected()` - https://angular.github.io/protractor/#/api?view=webdriver.WebElement.prototype.isSelected